src/pages/UserInfos/UserInfos.svelte

```diff
@@ -30,8 +31,16 @@
 
   const onSubmit = (): void => {
     schema.validate(userInfos).then(() => {
-      // TODO: Store data in firestore
-      console.log(userInfos)
+      const payload = {
+        age: userInfos.age,
+        sexe: userInfos.sexe,
+        anyExperience: userInfos.anyExperience,
+        experienceGrades: userInfos.experienceGrades.map((experienceGrade) => ({
+          labels: experienceGrade.labels,
+          grade: experienceGrade.grade
+        }))
+      }
+      api.addUserRequest(payload)
```

Comment:
Handle async until request done to show loading animation on button next. Maybe first do #53 and use it here.

Reply:
Ok